code/TestAutomation.Tests.BDD/Features/EpamSiteCareersPageTestsBDT.feature

```diff
@@ -0,0 +1,30 @@
+﻿Feature:EpamSiteCareersPageTestsBDT
+
+As a user interested in the Epam company
+I want to be able to reviw Careers page
+In order to be able to get information about opend roles in the Epam company
+
+
+@Smoke
+@CareersLokations
+Scenario: Epam Site - Careers Page - Locations
+	Given I navigate to the main page of Epam site
+	And I accept all cookies on Epam site
+	And I open Careers page with click on 'Careers' button 
+	When I click on the 'Find your dream job button'
```

Comment:
do not forget "Where" part of step :)
Example: I click on the 'Find your dream job button' on Careers page

Reply:
done